### DISCUSSION TITLE
AWS Connection for the AWS Secrets Manager backend

### DISCUSSION THREAD

**Discussion by nicolamarangoni** [apache/airflow#36713] [Q&A]
Hello,

I've successfully setup the AWS Secrets Manager as backend but currently I have only the connection named aws_default.
Is it possible to create a second AWS connection named aws_myconn and use it as backend?

Thanks,
Nicola

**Comment by Taragolis** [comment]
Secrets backends doesn't use Airflow Connections for establish connections, instead of that it expected configurations thought [`[secrets] backend_kwargs`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#backend-kwargs) configuration

Allowed options mentioned in [AWS Secrets Manager Backend](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/secrets-backends/aws-secrets-manager.html#aws-secrets-manager-backend) docs, see 

> [!TIP]
> To authenticate you can either supply arguments listed in [Amazon Webservices Connection Extra config](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html#howto-connection-aws-configuring-the-connection) or set [environment variables](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables).
> ```ini
> [secrets]
> backend = airflow.providers.amazon.aws.secrets.secrets_manager.SecretsManagerBackend
> backend_kwargs = {
>   "connections_prefix": "airflow/connections",
>   "variables_prefix": "airflow/variables",
>   "config_prefix": "airflow/config",
>   "role_arn": "arn:aws:iam::123456789098:role/role-name"
> }
> ```

**Reply by nicolamarangoni** [reply]
Ok thanks! It worked!
